website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -0,0 +1,75 @@
+# `invoke`
+
+```python
+def invoke(
+    contract_address: int,
+    function_name: str,
+    inputs: list[int] | dict[str, Any] | None = None,
+    max_fee: Optional[int] = None, # in Wei
+    auto_estimate_fee: bool = False,
+) -> InvokeResult:
+```
+See full documentation of InvokeResult [here](https://starknetpy.readthedocs.io/en/latest/contract.html?highlight=InvokeResult#starknet_py.contract.InvokeResult).
+Auto-fee estimation is supported, and `starknet.py`'s estimation logic is used - see [starknet.py docs](https://starknetpy.readthedocs.io/en/latest/guide.html?highlight=auto%20estimate#automatic-fee-estimation).
+
+:::warning
+Only sync methods are allowed inside of hints! (use `wait_for_acceptance_sync` instead of `wait_for_acceptance`)
+:::
```

Comment:
Why Protostar exposes incompatible API? This cheat code should be redesigned. 

`MigratorDeployContractCheatcode` expects `wait_for_acceptance` to be passed through config.

Reply:
This has been corrected